demisto_sdk/commands/format/update_layout.py

```diff
@@ -211,3 +219,44 @@ def layoutscontainer__arguments_to_remove(self):
                 second_level_args[kind] = set(self.data[kind].keys()) - set(kind_schema)
 
         return first_level_args, second_level_args
+
+    def remove_copy_and_dev_suffixes_from_layoutscontainer(self):
+        if name := self.data.get('name'):
+            self.data['name'] = remove_copy_and_dev_suffixes_from_str(name)
+
+        container = None
+        for kind in LAYOUTS_CONTAINER_CHECK_SCRIPTS:
+            if self.data.get(kind):
+                container = self.data.get(kind)
+                break
+        if container:
+            for tab in container.get('tabs'):
+                if tab.get('sections'):
+                    for section in tab.get('sections', ()):
+                        if section.get('queryType') == SCRIPT_QUERY_TYPE:
+                            section['query'] = remove_copy_and_dev_suffixes_from_str(section.get('query'))
+                            section['name'] = remove_copy_and_dev_suffixes_from_str(section.get('name'))
+
+    def remove_copy_and_dev_suffixes_from_layout(self):
+        if typename := self.data.get('TypeName'):
+            self.data['TypeName'] = remove_copy_and_dev_suffixes_from_str(typename)
+        if type_id := self.data.get('typeId'):
+            self.data['typeId'] = remove_copy_and_dev_suffixes_from_str(type_id)
+
+        layout_data = self.data.get('layout')
+        if layout_data:
+            layout_tabs = layout_data.get('tabs')
+            layout_sections = layout_data.get('sections')
+            if layout_tabs:
+                for tab in layout_tabs:
+                    if tab.get('sections'):
```

Comment:
this can be removed as the following `for` will not run if the condition is not satisfied (and there's no `else`)

Reply:
done